uniffi_bindgen/src/bindings/swift/templates/BridgingHeaderTemplate.h

```diff
@@ -32,7 +32,7 @@ typedef struct RustBuffer
 typedef int32_t (*ForeignCallback)(uint64_t, int32_t, const uint8_t *_Nonnull, int32_t, RustBuffer *_Nonnull);
 
 // Task defined in Rust that Swift executes
-typedef void (*UniFfiRustTaskCallback)(const void * _Nullable);
+typedef void (*UniFfiRustTaskCallback)(const void * _Nullable, int8_t);
```

Comment:
May we prefer a `uint8_t` here?

```suggestion
typedef void (*UniFfiRustTaskCallback)(const void * _Nullable, uint8_t);
```

Reply:
The reason we've historically used signed ints is because Kotlin only had experimental support (which was inherited from Java).  I think that Kotlin has dropped the "experimental" label, but I think I prefer sticking with signed ints for now.  Maybe in a future commit we can switch them all to unsigned.

Reply:
OK, I didn't know, thanks for the explanation.